### ISSUE TITLE
YAML editor breaks Git Merge Tool

### ISSUE THREAD

**Issue by j-ulrich** [de-jcup/eclipse-yaml-editor#88]
When I have the Yaml editor installed, the Git Merge Tool doesn't work for conflicting YAML files. Instead of seeing a comparison of the commits being merged, I just get a blank view:

![Bildschirmfoto 2020-04-21 um 12 22 21](https://user-images.githubusercontent.com/1260380/79855369-29bbe180-83cb-11ea-8632-29c8e0d577b3.png)
=>
![Bildschirmfoto 2020-04-21 um 12 16 18](https://user-images.githubusercontent.com/1260380/79855968-fded2b80-83cb-11ea-8c49-6ec17ea0cd1e.png)

The normal compare view works although it doesn't do any YAML syntax highlighting.

I am using Yaml Editor version 1.6.2 with Eclipse IDE for C/C++ Developers version 2019-09 R (4.13.0).

**Comment by de-jcup** [comment]
Hello @j-ulrich ,

I tried to reproduce the problem:
- I created a testproject
- initialized a git repo
- created a yaml file
- committed the file to git 
- changed something in file
- added to index
- doubleclicked on local version (as you described)

But it is working as expected:
![image](https://user-images.githubusercontent.com/11644753/80893485-051d0f00-8cd3-11ea-85b3-034642622b7b.png)

For the test I used eclipse-2020-03 (Java IDE) at my Linux machine. At work I currently use 2019-09 (like you do) and had never such issues (also Linux). After I was not able to reproduce, I installed newest cpp eclipse package, installed yaml-editor by marketplace and tried out with cpp eclipse. Same result.

Now I installed cpp eclipse at a windows machine
![image](https://user-images.githubusercontent.com/11644753/80894371-bbd0bd80-8cda-11ea-8585-4463fca04580.png)

so  still there is all working - 
So I am not able to reproduce the problem.
